test/extensions/filters/http/rbac/rbac_filter_test.cc

```diff
@@ -129,6 +162,23 @@ TEST_F(RoleBasedAccessControlFilterTest, Path) {
       {":authority", "host"},
   };
   EXPECT_EQ(Http::FilterHeadersStatus::Continue, filter_.decodeHeaders(headers, false));
+
+  headers = Http::TestRequestHeaderMapImpl{
+      {":method", "GET"},
+      {":path", "prefix/suffix/next"},
+      {":scheme", "http"},
+      {":authority", "host"},
+  };
+  EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, filter_.decodeHeaders(headers, false));
+
+  // Check Log
+  setMetadata();
```

Comment:
same here, move the setMetadata() to the above, and remove the extra EXPECT_EQ() in the next line.